interfaces/hotplug/deviceinfo.go

```diff
@@ -0,0 +1,77 @@
+// -*- Mode: Go; indent-tabs-mode: t -*-
+
+/*
+ * Copyright (C) 2018 Canonical Ltd
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License version 3 as
+ * published by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ */
+
+package hotplug
+
+import (
+	"path/filepath"
+
+	"github.com/snapcore/snapd/dirs"
+)
+
+// HotplugDeviceInfo carries information about added/removed device detected at runtime.
+type HotplugDeviceInfo struct {
+	object string
```

Comment:
Can you just call this `devpath` please, here and everywhere in the hot plug code. My desire here is to limit the vocabulary used so that we don't invent names for existing kernel constructs.

Reply:
Hmmm, I could, however this corresponds to KObj (kernel object?) value associated with the udev event reported by go-udev (what I get from go-udev is kobj string and map of extra data such as DEVPATH etc). While there seems to be direct relation between kobj and DEVPATH, I'm not 100% sure it's always the case, so perhaps it's ok to leave the doors open in case we find both devpath and object values useful?

Reply:
Either `devpath` and `kobjsomething` are better than `object` which is as generic as one can get. Whatever you do please explain what the value is in a comment.